.alexrc

```diff
@@ -1,3 +1,3 @@
 {
-  "allow": ["color", "dad-mom", "her-him", "he-she", "hook", "hooks", "harder"] 
+  "allow": ["color", "dad-mom", "disabled", "her-him", "he-she", "hook", "hooks", "harder"] 
```

Comment:
do we want to allow this even though the content was updated so that it doesn't need to be allowed?

Reply:
you're right, I should remove it.